apollo-gradle-plugin-external/src/main/kotlin/com/apollographql/apollo3/gradle/api/Service.kt

```diff
@@ -511,6 +511,18 @@ interface Service {
    */
   val sealedClassesForEnumsMatching: ListProperty<String>
 
+  /**
+   * The annotation to use for experimental fields/inputFields/enumValues
+   *
+   * You can pass the special value "none" to disable adding an annotation.
+   * If you're using a custom annotation, it must be able to target:
+   * - AnnotationTarget.PROPERTY
+   * - AnnotationTarget.CLASS
+   *
+   * Default: "com.apollographql.apollo3.annotations.ApolloExperimental"
```

Comment:
I'm not 100% convinced we should re-use `ApolloExperimental` for the default as it is currently kind of specific to the project's codebase, so maybe it would be worth it to introduce a specific `Experimental` annotation.
Not a big deal for the time being!

Reply:
Agreed the semantics are different but didn't find a good alternative name. `GraphQLExperimental` maybe ?

Reply:
I was thinking just `Experimental` but I like `GraphQLExperimental` too. Another idea: `SchemaExperimental`.

Reply:
I was a bit concerned `Experimental` would clash with other annotation but maybe it's not the case. Maybe we should make `@Experimental` `@ApolloExperimental` 🙃 . 

Just kidding. I'll go with `@Experimental` and we'll see how that unfolds.